projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -301,30 +327,28 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
    *
    * @param datePart The optional DatePart to increment. Defaults to Date or Hours(when Date is absent from the inputFormat - ex:'HH:mm').
    */
-  public increment(datePart?: DatePart): void {
+  public increment(datePart?: DatePart, spinDeltas?: DatePartDeltas): void {
```

Comment:
Considering the delta can be set via the public property, why would you be providing the full set here for this method as well? At the very least, since increment/decrement methods only work with a since part at a time, if there's a secondary setting for delta it should be just the numeric delta for that part, not the whole thing, no?